__tests__/components/lists/FeedList.test.tsx

```diff
@@ -0,0 +1,37 @@
+import { NavigationContainer } from '@react-navigation/native';
+import { render } from '@testing-library/react-native';
+import React from 'react';
+import FeedList from '../../../components/lists/FeedList';
+import { Feed as FeedData } from '../../../mock/feed';
+
+jest.mock('expo-router', () => ({
+  Link: ({ children }: any) => children, // Mock Link component to render its children directly
+}));
```

Comment:
We should type this properly instead of using `any`, and remove the comments in this file, they do not add much context.

```ts
jest.mock('expo-router', () => ({
  Link: ({ children }: { children: React.ReactNode }) => children,
}));
```

Reply:
also if you're not testing the navigation, its moot.